docs/standard/tour.md

```diff
@@ -14,137 +14,130 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
+Async programming is a first-class concept within .NET with async support in the runtime, framework libraries, and .NET language constructs. Internally, they're based off of objects (such as `Task`), which take advantage of the operating system to perform I/O-bound jobs as efficiently as possible.
 
-To learn more about async programming in .NET, start with the [Async overview](async.md).
+To learn more about async programming in .NET, start with the [Async overview](async.md) topic.
 
 ## Language Integrated Query (LINQ)
 
-LINQ is a powerful set of features for C# and VB that allow you to write simple, declarative code for operating on data. The data can be in many forms (such as in-memory objects, in a SQL database, or an XML document), but the LINQ code you write typically won't look different for each data source!
+LINQ is a powerful set of features for C# and VB that allow you to write simple, declarative code for operating on data. The data can be in many forms (such as in-memory objects, in a SQL database, or an XML document), but the LINQ code you write typically won't look different for each data source.
 
-To learn more and see some samples, check out [LINQ (Language Integrated Query)](using-linq.md).
+To learn more and see some samples, see the [LINQ (Language Integrated Query)](using-linq.md) topic.
 
 ## Native interoperability
 
-Every operating system in current use provides a lot of platform support for various programming tasks. .NET provides several ways to tap into those APIs. Collectively, this support is called "native interoperability" and in this section we will take a look at how to access native APIs from managed, .NET code.
+Every operating system in current use provides a lot of platform support for various programming tasks. .NET provides several ways to tap into those APIs. Collectively, this support is called *native interoperability*; and in this section, accessing native APIs from managed .NET code is examined.
 
-The main way to do native interoperability is via "platform invoke" or P/Invoke for short. This support in .NET Core is available across Linux and Windows platforms. Another, Windows-only way of doing native interoperability is known as "COM interop" which is used to work with [COM components](https://msdn.microsoft.com/library/bwa2bx93.aspx) in managed code. It is built on top of P/Invoke infrastructure, but it works in subtly different ways.
+The main way to do native interoperability is via "platform invoke" or P/Invoke for short. This support in .NET Core is available across Linux and Windows platforms. Another Windows-only way of doing native interoperability is known as "COM interop," which is used to work with [COM components](https://msdn.microsoft.com/library/bwa2bx93.aspx) in managed code. It's built on top of P/Invoke infrastructure, but it works in subtly different ways.
```

Comment:
instead of "this support in..." --> ", which is supported across"
"Another Windows-only" --> "A Windows-only"
"of P/Invoke" -> of the P/Invoke